README.md

```diff
@@ -56,9 +56,11 @@ tools:
 
 ## Community
 
-* GitHub Discussions: see [`GitHub Discussions`](https://github.com/JanusGraph/janusgraph/discussions) 
+* GitHub Discussions: see [`GitHub Discussions`](https://github.com/JanusGraph/janusgraph/discussions)
   for all general discussions and questions about JanusGraph
-  
+
+* Discord for interactive discussions and questions about JanusGraph: [Join the server](https://discord.gg/5n4fjv4QAf)
+
 * Chat rooms on Gitter:
```

Comment:
Should Gitter be removed now that Discord is up?

Reply:
That's a good question. @porunov [suggested on janusgraph-dev](https://lists.lfaidata.foundation/g/janusgraph-dev/message/1575) that we keep Gitter for now as we have 864 members there and wait until Discord has a similar popularity before we abandon it.
Now we could of course keep Gitter active but stop promoting it in the README & the issue template (I just noticed that I forgot to add Discord there as well, I'll add that).
I personally think we should wait a bit to see whether users and contributors actually want to use Discord and whether the experience is better on Discord vs Gitter (e.g., are questions from users actually answered) before we completely remove all mentions of Gitter.

Reply:
Updated the commit & PR description.